Source/ModelGroups/Node_Attribute.cs

```diff
@@ -1,167 +1,136 @@
             }
 
-            foreach (Property property in combo)
+            void SetTranslationY(List<Property> properties, Runtime.Node node)
             {
-                if (property.name == Propertyname.Matrix)
-                {
-                    nodeList[1].Matrix = specialProperties[0].value;
-                }
-                else if (property.name == Propertyname.Translation ||
-                         property.name == Propertyname.Translation_X ||
-                         property.name == Propertyname.Translation_Y ||
-                         property.name == Propertyname.Translation_Z)
-                {
-                    nodeList[1].Translation = property.value;
-                }
-                else if (property.name == Propertyname.Rotation)
-                {
-                    nodeList[1].Rotation = specialProperties[1].value;
-                }
-                else if (property.name == Propertyname.Scale)
-                {
-                    nodeList[1].Scale = property.value;
-                }
+                node.Translation = new Vector3(0, 2, 0);
+                properties.Add(new Property(PropertyName.Translation, node.Translation));
+            }
+
+            void SetTranslationZ(List<Property> properties, Runtime.Node node)
+            {
+                node.Translation = new Vector3(0, 0, -2);
+                properties.Add(new Property(PropertyName.Translation, node.Translation));
+            }
+
+            void SetRotation(List<Property> properties, Runtime.Node node)
+            {
+                var rotation = Quaternion.CreateFromAxisAngle(new Vector3(0f, 1f, 0f), (float)Math.PI);
+                rotation.W = (float)Math.Round(rotation.W);
+                node.Rotation = rotation;
+                properties.Add(new Property(PropertyName.Rotation, rotation));
             }
 
-            // Apply the material to each node
-            foreach (var node in nodeList)
+            void SetScale(List<Property> properties, Runtime.Node node)
             {
-                node.Mesh.MeshPrimitives[0].Material = material;
+                node.Scale = new Vector3(1.2f, 1.2f, 1.2f);
+                properties.Add(new Property(PropertyName.Scale, node.Scale));
             }
 
-            return wrapper;
+            void SetMatrix(List<Property> properties, Runtime.Node node)
+            {
+                var matrixT = Matrix4x4.CreateTranslation(new Vector3(-2, 2, -2));
+                var matrixR = Matrix4x4.CreateFromAxisAngle(new Vector3(0f, 1f, 0f), (float)(Math.PI));
+                var matrixS = Matrix4x4.CreateScale(1.2f);
+                var matrixTRS = Matrix4x4.Multiply(Matrix4x4.Multiply(matrixS, matrixR), matrixT);
+                node.Matrix = matrixTRS;
+                properties.Add(new Property(PropertyName.Matrix, matrixTRS));
+            }
+
+            this.Models = new List<Model>
+            {
+                CreateModel((properties, node) => {
+
```

Comment:
This looks like an error. Probably good to put a comment here that it is intentionally blank. I usually put `// do nothing`

Reply:
Good idea. I'll do this in the other model groups as well.

Reply:
Fixed for this Mode group. I'll do the others in another PR.